cite.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <title>The ASPECT mantle convection code: How to cite?</title>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+    <link rel="icon" href="/favicon.ico"/>
+    <link rel="apple-touch-icon" href="/pictures/clover57.png"/>
+    <meta name="author" content="the deal.II authors"/>
+    <meta name="keywords" content="mantle convection, dealii, finite elements, finite element method, fem, finite element software, open source, adaptive, finite element method, finite element modeling, finite element modelling, finite element methods, numerical analysis, numerical analysis software, computational fluid dynamics software"/>
+    <meta name="robots" content="index, follow">
+
+    <!-- Include stylesheets -->
+    <link href="css/bootstrap.min.css" type="text/css" rel="stylesheet"/>
+    <link href="css/global.less" type="text/less" rel="stylesheet"/>
+    <link href="css/index.less" type="text/less" rel="stylesheet"/>
+
+    <!-- LESS needs to come after the stylesheets -->
+    <script src="js/less-1.4.1.min.js"></script>
+  </head>
+
+  <body>
+
+    <!--#include file="header.include" -->
+
+    <div id="content" class="container">
+      <div class="row">
+        <div class="well">
+
+            <h1>
+              Referencing <abbr>ASPECT</abbr>
+            </h1>
+
+	              <p>
+              As with all scientific work, funding agencies have a reasonable expectation
+              that if we ask for continued funding for this work, we need to demonstrate
+              relevance. To this end, we ask that if you publish results that were obtained
+              to some part using <abbr>ASPECT</abbr>, you give credit in the following ways:
+	      </p>
+
+            <p><b>1. Body text</b>
+	      <pre><code>
+Computations were done using the ASPECT code version 1.5.0, see \cite{heister_aspect_methods2,...}.</code></pre>
+	    </p>
+
+	    <p><b>2. Acknowledgement</b>
+	      <pre><code>
+We thank the Computational Infrastructure for Geodynamics (geodynamics.org) which is funded by the National Science Foundation under award EAR-0949446 and EAR-1550901 for supporting the development of ASPECT.</code></pre>
+	    </p>
+
+	    <p><b>3. Citation</b><br>
+	      Please cite one of all of the following documents. The canonical reference for this software:
```

Comment:
one _or_ all

Reply:
And can you put the sentence "The canonical reference for this software" into a new line?